pig/src/main/java/org/apache/iceberg/pig/PigParquetReader.java

```diff
@@ -60,9 +60,20 @@
 import org.apache.pig.data.DataByteArray;
 import org.apache.pig.data.Tuple;
 import org.apache.pig.data.TupleFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+/**
+ * @deprecated will be removed in 1.8.0
+ */
+@Deprecated
 public class PigParquetReader {
-  private PigParquetReader() {}
+
+  private static final Logger LOG = LoggerFactory.getLogger(PigParquetReader.class);
+
+  private PigParquetReader() {
+    LOG.warn("Iceberg Pig is not deprecated. It will be removed in Iceberg 1.8.");
```

Comment:
"not deprecated"? should be deprecated?

Reply:
same comment for all the places. 

I too feel this logging may be unnecessary.

Reply:
It's "now" not "not": a typo. I'm fixing it.